Update call-out shortcode usage in Contributing (#286)
nginx/documentation: CONTRIBUTING_DOCS.md

- `caution`
- `warning`

You can also create custom callouts using the `call-out` shortcode `{{< call-out "type position" "header" "font-awesome icon >}}`. For example:

```md
{{<call-out "important side-callout" "JWT file required for upgrade" "fa fa-exclamation-triangle">}}
```

> By default, all custom callouts are inlined unless `side-callout` is passed in to put the callout on the right of the content.

Reviewer: ```suggestion
By default, all custom callouts are included inline, unless you add `side-callout` which places the callout to the right of the content.
```

Reviewer: Non-blocking suggestion
